deploy/00_resolve_keep_registry.ts

```diff
@@ -11,10 +11,15 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   if (KeepRegistry && helpers.address.isValid(KeepRegistry.address)) {
     log(`using existing KeepRegistry contract at ${KeepRegistry.address}`)
   } else if (
-    hre.network.name !== "hardhat" ||
+    hre.network.name === "hardhat" &&
     (hre.network.config as HardhatNetworkConfig).forking.enabled
   ) {
     throw new Error("deployed KeepRegistry contract not found")
+  } else if (
+    hre.network.name !== "hardhat" &&
+    hre.network.name !== "development"
```

Comment:
`hardhat` and `development` networks are tagged `local` in the `hardhat.config.ts`. We could use `hre.network.tags.local` here.
```suggestion
    !hre.network.local
```

Reply:
pls see these comments https://github.com/keep-network/tbtc-v2/pull/365#discussion_r897137774 
We already use hardhat's configuration in CI which has a `local` tag under the hood, but at the same time, CI is not a local machine so writing `!hre.network.local` when it runs on CI also doesn't hold 100%.

Reply:
I would prefer to look at the network name in this condition. "local" tag is ambiguous. We may need more conditions for internal testnet and public testnet.

Reply:
We will use `allowStubs` tag.